Network/Nmcli.hs

```diff
@@ -27,7 +27,7 @@ scanCmd = Scan "nmcli --terse --fields ssid,signal dev wifi"
 
 -- | Command to list the wifi the computer can currently auto connect to
 knownCmd :: Command
-knownCmd = Scan "nmcli --terse --fields name con list"
```

Comment:
Unfortunately, I have no more mint nor ubuntu.
I can check on nixos though (but not right now).